### PR TITLE
Move berta editor in to iframe

### PR DESCRIPTION
**Pull request by vteivans**
- [x] `/engine` contains iframe with the editor instead of editor itself
- [x] New endpoint with editor created (`/engine/editor`)
- [x] PHP path variables updated to work with the new engine
    * Path and URL variables separated
    * Path and URL  variables renamed to better represent their use
    * Docs added for URL variables
- [x] Bring the menu out of the the iframe, separate it from the page
- [x] Fix creating new sites first section
- [x] Fix Tips and Tricks popup: `berta/engine/js/BertaEditorBase.js:120`

### CODE REVIEW

File: engine/loader.helper.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<?php
+require_once __dir__. '/../class-loader/ClassLoader.php';
```
Comment (uldisrudzitis) [inline engine/loader.helper.php]: Still getting include error in public page and in `/engine`:
```
Fatal error: require_once(): Failed opening required '/Applications/MAMP/_projects/berta/engine/../class-loader/ClassLoader.php'
```

File: engine/editor/inc.editor.php [hunk 2]
```diff
@@ -0,0 +1,73 @@
+<?php
+/** THis file can not be loaded directly through browser, it's only meant to be included */
+$included_files = get_included_files();
+if (!in_array(__FILE__, get_included_files()) || $included_files[0] == __FILE__) {
+    /* IF this file is not included, redirect to root of the page */
+    return header('Location: /');
```
Comment (uldisrudzitis) [inline engine/editor/inc.editor.php]: No need to `return` here.

File: engine/error/404.php [hunk 3]
```diff
@@ -4,14 +4,14 @@
 <html>
 <head>
     <meta charset="UTF-8">
-	<title><?php echo $berta->settings->get('texts', 'page-title') ?></title>
-	<link rel="SHORTCUT ICON" href="<?php echo $SITE_ABS_ROOT ?>favicon.ico">
-	<link rel="stylesheet" href="<?php echo $ENGINE_ABS_ROOT ?>css/id.css.php" type="text/css">
+    <title><?php echo $berta->settings->get('texts', 'page-title') ?></title>
+    <link rel="SHORTCUT ICON" href="<?php echo $SITE_ROOT_URL ?>favicon.ico">
```
Comment (uldisrudzitis) [inline engine/error/404.php]: Add `<?php echo $SITE_ROOT_URL ?>` in other files where favicon is used.

**Comment by uldisrudzitis** [comment]
Top menu panel is not visible when editing section with type `shopping cart`

**Comment by uldisrudzitis** [comment]
When switching to other site from multisite menu using url param `?site=second` would be nice if `Main control panel menu` knows about that otherwise it's not possible to edit other site settings.

**Comment by vteivans** [comment]
@uldisrudzitis can't repeat, but I have issues with new multipage section setup.

**Pull request merged**